catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/reader/FeatureCollectionMessageBodyReaderWfs20.java

```diff
@@ -176,7 +176,7 @@ public Wfs20FeatureCollection readFrom(
         throw new WebApplicationException(e1, response);
       } catch (JAXBException | XMLStreamException e1) {
         LOGGER.debug("Error in retrieving feature collection.", e1);
-      } catch (RuntimeException | Error e) {
+      } catch (Exception e) {
```

Comment:
❓  Curious what the original reason was to catch `Error` and why it's no longer necessary. This change definitely demands justification.